src/components/Table.jsx

```diff
@@ -0,0 +1,94 @@
+import { Button } from '@mui/material';
+import { DataGrid } from '@mui/x-data-grid';
+import data from './data.json';
+
+const multiplePublishButtons = (params) => {
+  return (
+    <>
+       <Button
+        variant="outlined"
+        color="primary"
+        size="small"
+        style = {{ marginRight: "7px" }}
```

Comment:
Please use the spacing system from MUI in order to add space between elements
https://mui.com/system/spacing/